demisto_sdk/commands/common/content/content.py

```diff
@@ -53,16 +54,13 @@ def from_cwd(cls) -> Content:
         TODO:
             1. Add attribute which init only changed objects by git.
         """
-        repo = cls.git()
-        if repo:
-            content = Content(repo.working_tree_dir)  # type: ignore
-        else:
-            content = Content(Path.cwd())
-
-        return content
+        try:
+            return Content(str(cls.git().repo.working_tree_dir))
+        except InvalidGitRepositoryError:
+            return Content(Path.cwd())
 
     @staticmethod
-    def git() -> Repo | None:
+    def git() -> GitUtil:
```

Comment:
`git()` looks odd. It would be clearer as a class attribute, I suggest taking the logic into a separate function outside the class, then in the class level doing `git_util = create_git_util()`

Reply:
even though i think you are right, ill change the name of the method to be `git_util`. 
I think that once we refactor the validate the code here will be completely irrelevant, i don't see the point in maintaining it much further